packages/core/util/blockTypes.ts

```diff
@@ -96,6 +98,7 @@ export class BaseBlock {
     this.start = data.start
     this.end = data.end
     this.key = data.key
+    this.offsetPx = data.offsetPx
```

Comment:
was offsetPx always being passed to the block constructors, and just not being saved? I don't see any changes to the block creation

Reply:
I believe so, there is an Object.assign(this,args) which uses that, but now I access the offsetPx. The code on the LGV doesn't manually position the blocks (LinearBlocks only positions a set of blocks at a time but the svg has to position every block)